virtual_rainforest/models/animals/animal_communities.py

```diff
@@ -185,6 +206,11 @@ def forage_community(self) -> None:
         plant_list = [plant_community]
 
         for consumer_cohort in self.all_animal_cohorts:
+            if (
+                consumer_cohort.individuals == 0
+            ):  # temporary while finalizing die_cohort placements
+                continue
```

Comment:
I suppose you could print a warning here for now, so you don't forget to remove it.